packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -70,36 +72,36 @@ export class CustomScrollbar extends Component<Props> {
     this.updateScroll();
   }
 
-  renderTrack = (track: 'track-vertical' | 'track-horizontal', hideTrack: boolean | undefined, passedProps: any) => {
+  renderTrack = (styles: any, hideTrack: boolean | undefined, passedProps: any) => {
     if (passedProps.style && hideTrack) {
       passedProps.style.display = 'none';
     }
 
-    return <div {...passedProps} className={track} />;
+    return <div {...passedProps} className={styles.trackVertical | styles.trackHorizontal} />;
```

Comment:
Are you sure about this ternary? Aren't you're already passing the correct styles to this function? I think if you add a proper type to styles you'd get a transpile error here.

Reply:
wow...would fix it right away.
Thanks for bringing this up!